projects/packages/schema/tests/php/integration/Integration_Serialization_Test.php

```diff
@@ -93,7 +93,7 @@ public function test_serialization_to_string() {
 		);
 
 		$this->assertSame( $expect_with_fallback, $with_fallback->schema() );
-		$this->assertSame( wp_json_encode( $expect_with_fallback ), wp_json_encode( $with_fallback ) );
+		$this->assertSame( wp_json_encode( $expect_with_fallback, JSON_UNESCAPED_SLASHES ), wp_json_encode( $with_fallback, JSON_UNESCAPED_SLASHES ) );
```

Comment:
I wonder why this doesn't compare the two objects directly. 🤷

Anyway, if you want, we could probably include `JSON_UNESCAPED_UNICODE`

```suggestion
		$this->assertSame( wp_json_encode( $expect_with_fallback, JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE ), wp_json_encode( $with_fallback, JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE ) );
```

Reply:
If I'm reading the docs correctly, I believe `assertSame()` only works if the two vars reference the same object. Perhaps `assertEquals()` would be better? Though it's unclear if that is a deep comparison or not.

Reply:
Oh, now that I look closer, maybe it's testing the `jsonSerialize()` behavior, versus the line above that's testing `->schema()`? Anyway, we could still do `JSON_UNESCAPED_UNICODE` here if you want.